cash-management-pool/lending.md

```diff
@@ -9,6 +9,7 @@ The borrower is only permitted to hold short-dated US Treasury bills reverse rep
 The pool will pass yield sourced from US Treasury bills and government-backed instruments, less fees of 0.5%, to Lenders. The APY will fluctuate with the Federal Reserve Rate and as Treasury bills mature and roll into new issuances.
 
 **Are the fees fixed?**
+
```

Comment:
note you need to use a `<br>` breakpoint if you want to add a new line in markdown